service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,201 @@
+                <econdition field-name="orderId"/>
+                <econdition field-name="orderPartSeqId"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!--  8.Find all Order Part records assigned to the facility with ZIRET_WH value-->
+
+    <service verb="get" noun="OrderPartData">
+        <in-parameters>
+            <parameter name="facilityId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId"/>
+            <parameter name="orderPartSeqId"/>
+            <parameter name="facilityId"/>
+            <parameter name="partName"/>
+            <parameter name="customerPartyId"/>
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="mantle.order.OrderPart" value-field="order"/>
+            <set field="orderId" from="order.orderId"/>
+            <set field="orderPartSeqId" from="order.orderPartSeqId"/>
+            <set field="statusId" from="order.statusId"/>
+            <set field="vendorPartyId" from="order.vendorPartyId"/>
+            <set field="customerPartyId" from="order.customerPartyId"/>
+            <set field="facilityId" from="order.facilityId"/>
+            <set field="carrierPartyId" from="order.carrierPartyId"/>
+            <set field="shipmentMethodEnumId" from="order.shipmentMethodEnumId"/>
+            <set field="partTotal" from="order.partTotal"/>
+            <set field="priority" from="order.priority"/>
+            <set field="disableShippingCalc" from="order.disableShippingCalc"/>
+            <set field="disableTaxCalc" from="order.disableTaxCalc"/>
+        </actions>
+    </service>
+
+    <!--   9.Finding the count of order parts for the customer Party Id, “CustJqp”.-->
+
+    <service verb="get" noun="OrderPartsCount">
+        <in-parameters>
+            <parameter name="customerPartyId"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderPartSeqId" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find-count entity-name="mantle.order.OrderPart">
+                <econdition field-name="customerPartyId"/>
+            </entity-find-count>
+        </actions>
+    </service>
+
+    <!--10.Finding all unique Order Parts with shipmentMethodEnumId value as “ShtMthGround” and facilityId as “ZIRET_WH”.-->
+
+    <service verb="get" noun="OrderPartsInfo">
+        <in-parameters>
+            <parameter name="shipmentMethodEnumId"/>
+            <parameter name="facilityId"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderPartsList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="orderPartsList"/>
+            <set field="products" from="[]"/>
+            <iterate list="orderPartsList" entry="product">
+                <if condition="shipmentMethodEnumId=='ShtMthGround' &amp;&amp; facilityId=='ZIRET_WH' ">
```

Comment:
This condition should not be added like this, we can directly add in entity-find, please check and update to find the distinct records.